modules/ROOT/pages/basic-authentication-simple-concept.adoc

```diff
@@ -8,26 +8,31 @@ endif::[]
 >s| Policy Name | Basic Authentication: Simple
 >s|Summary      | Allows access based on the basic authorization mechanism, with a single user-password
 >s|Category |
->s| Since Mule Version | 4.0.0
+>s| Available from Mule version | 4.0.0
 .1+>.^s| Returned Status Codes
```

Comment:
Policy returns 401 when invalid username/password are provided. As per RFC, when this status code is returned the policy sends back also a "WWW-Authenticate" HTTP header field with the following format: 

`WWW-Authenticate: Basic realm="mule-realm"`

Reply:
Also, the name of the policy is not "Basic Authentication: Simple", but "Basic Authentication - Simple".

Reply:
Done! Thanks!